internal/resources/metal/organization_member/models.go

```diff
@@ -0,0 +1,58 @@
+package organizationmember
+
+import (
+	"context"
+	"path"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+)
+
+type ResourceModel struct {
+	ID             types.String `tfsdk:"id"`
+	Invitee        types.String `tfsdk:"invitee"`
+	InvitedBy      types.String `tfsdk:"invited_by"`
+	OrganizationID types.String `tfsdk:"organization_id"`
+	ProjectsIDs    types.Set    `tfsdk:"projects_ids"`
+	Nonce          types.String `tfsdk:"nonce"`
+	Message        types.String `tfsdk:"message"`
+	Created        types.String `tfsdk:"created"`
+	Updated        types.String `tfsdk:"updated"`
+	Roles          types.Set    `tfsdk:"roles"`
+	State          types.String `tfsdk:"state"`
+}
+
+// func (m *ResourceModel) parse(ctx context.Context, invitee string, members []packngo.Member, invitations []packngo.Invitation) diag.Diagnostics {
+func (m *ResourceModel) parse(ctx context.Context, member *member) diag.Diagnostics {
+	var diags diag.Diagnostics
+
+	if member.isMember() {
+		projectsList, _ := types.SetValueFrom(ctx, types.StringType, member.Member.Projects)
```

Comment:
We should capture and respond to all of these potential errors in calls to `SetValueFrom`
```suggestion
		projectsList, diag := types.SetValueFrom(ctx, types.StringType, member.Member.Projects)
		
		if diag.HasError() {
		  return diag
		}
```